packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -170,15 +170,19 @@ export class SingleAddressWallet implements ObservableWallet {
       connectionStatusTracker$ = createSimpleConnectionStatusTracker()
     }: SingleAddressWalletDependencies
   ) {
-    this.#logger = logger;
+    this.#logger = contextLogger(logger, 'SingleAddressWallet');
```

Comment:
I think we should keep it consistent. Right now,
1. SingleAddressWallet derives contextLogger by itself
2. createProviderStatusTracker expects to get a logger that is already derived

I think strategy 2. is a little cleaner in terms of code. How about we change this to:

```this.#logger = contextLogger(logger, `(${this.name})`);```

So that it's a different reason to derive the logger here.

Reply:
Agree.
Why do you want to wrap it in parentheses? It will end up looking like this: [(My cool wallet)]

Reply:
No need to wrap it then

Reply:
Done in [3eeba2a](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/3eeba2a461134843d202116d39c1edec2c118116)